api/nakadi-event-bus-api.yaml

```diff
@@ -871,6 +871,35 @@ paths:
           description: Unprocessable Entity
           schema:
             $ref: '#/definitions/Problem'
+    patch:
+      tags:
+        - subscription-api
+      security:
+        - oauth2: ['nakadi.event_stream.read']
+      description: |
+        Endpoint for reseting subscription offsets.
+      parameters:
+        - name: subscription_id
+          in: path
+          type: string
+          description: Id of subscription
+          required: true
+        - name: cursors
+          in: body
+          schema:
+            type: object
+            properties:
+              items:
+                description: |
+                  List of cursors to reset to.
+                type: array
+                items:
+                  $ref: '#/definitions/SubscriptionCursor'
```

Comment:
I think it makes sense to use `SubscriptionCursorWithoutToken` here instead, because user will not really have a token when resetting cursors.